lib/sass_spec/runner.rb

```diff
@@ -42,7 +42,18 @@ def run
       Minitest.reporter = Minitap::TapY
     end
 
-    Minitest.run(minioptions)
+    result = Minitest.run(minioptions)
+
+    if @options[:probe_todo]
+      puts "Reporting successfully probed todo tests"
```

Comment:
Here I access the stored result to determine which todo tests did pass.
The ones that did not pass should be ignored as if they were skipped.